app/controllers/needs_controller.rb

```diff
@@ -47,6 +46,7 @@ def update_multiple
     for_update = JSON.parse(params[:for_update])
     for_update.each do |obj|
       need = Need.find(obj['need_id'])
+      authorize(@need)
```

Comment:
This doesn't look right - should be `need`

Reply:
Good spot, this is a mistake.  Fixed